packages/react-form-renderer/src/files/use-field-api.js

```diff
@@ -76,6 +76,9 @@ const useFieldApi = ({ name, initializeOnMount, component, render, validate, ...
   };
 
   const fieldProps = useField(name, enhancedProps);
+  if (fieldProps.input.type === 'file' && typeof fieldProps.input.value === 'object') {
```

Comment:
I think this modification should be moved to the return statement of this hook.

Reply:
why?

Reply:
With this the input object is basically being modified at two different places. I would keep it at one place - it would be more consistent and readable.

Reply:
its not going to be pretty. But ok